data/paracosma-nepal.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Paracosma Nepal",
+    "establishedAt": 1451589361,
+    "location": "naxal, Kathmandu",
+    "website": "https://www.paracosma.com/",
+    "email": null,
+    "phone": "+977-1-4004778",
+    "socials": {
+      "facebook": "https://www.facebook.com/paracosmaInc",
+      "instagram": "https://www.instagram.com/paracosma_studio/",
+      "github": null,
+      "linkedin": "https://www.linkedin.com/company/paracosma/",
+      "twitter": "https://twitter.com/Paracosmainc",
+      "youtube": "https://www.youtube.com/channel/UCf4_ArJPd3vT2FmErBbEcUA"
+    }
+  }
+  
```

Comment:
There seems some formatting issue at the end of the line. 👀 Please refer to other json files once and adjust any spaces or new lines accordingly. 🙏